src/main/java/org/mule/module/json/internal/JsonSchemaValidator.java

```diff
@@ -203,7 +213,9 @@ public JsonSchemaValidator build() {
           .freeze();
 
       try {
-        return new JsonSchemaValidator(loadSchema(factory), objectMapper);
+        checkArgument(!(isBlank(schemaContent) && isBlank(schemaLocation)), "Schema cannot be null or blank");
```

Comment:
Here doesn' t check if both two fields are provided. 
User has to choose only one field to input. If they provide both fields, we should show error.

Reply:
Modify the exception handling for these cases. Evaluating that both are null. Also that both have value.